### ISSUE TITLE
Pop-Up "Paste as HTML"

### ISSUE THREAD

**Issue by sameersmalsus** [xdan/jodit#1156]
**Jodit Version:** 4.1.2

**Browser:** all
**OS:** Windows
**Is React App:** true
**Reproduced on xdsoft.net:** true

**Code**

```js
Can we Change the Modal Content on add more buttons in this modal ?
```

**Expected behavior:**

**Actual behavior:**
![Screenshot 2024-07-05 132501](https://github.com/xdan/jodit/assets/119491663/31f16fec-60a4-47ce-9978-d26ba047ae79)

**Comment by Liteb0t** [comment]
I don't quite understand the question. I'm guessing the problem is it always gives that pop-up when copy-pasting styled text, which is annoying.

**Comment by jaspersmedia** [comment]
Would be nice if there are more shortcuts: CTRL SHIFT V --> Plain paste for example.

**Comment by Liteb0t** [comment]
CTRL SHIFT V is already taken by paste storage plugin

**Comment by xdan** [comment]
Yes — the buttons in the *Paste as HTML* dialog are configurable via the **`pasteHTMLActionList`** option (and `pasteFromWordActionList` for the Word dialog):

```js
Jodit.make('#editor', {
  pasteHTMLActionList: [
    { value: Jodit.constants.INSERT_AS_HTML, text: 'Keep' },
    { value: Jodit.constants.INSERT_AS_TEXT, text: 'Insert as Text' },
    { value: Jodit.constants.INSERT_ONLY_TEXT, text: 'Insert only Text' }
  ]
});
```

Each entry is `{ value, text }` — `value` is the insert mode applied when the button is clicked, `text` is the label. Reorder, remove, or relabel entries as needed. To add a fully custom action with your own behavior, listen for the paste via `beforePaste`/`processPaste` events and handle it yourself instead of using the built-in dialog. Closing as answered — reopen if you need a specific custom action wired in.
